bundles/io.openliberty.tools.eclipse.ui/META-INF/MANIFEST.MF

```diff
@@ -7,12 +7,21 @@ Bundle-SymbolicName: io.openliberty.tools.eclipse.ui;singleton:=true
 Bundle-Version: 23.0.9.qualifier
 Bundle-Activator: io.openliberty.tools.eclipse.LibertyDevPlugin
 Export-Package: io.openliberty.tools.eclipse;x-friends:="io.openliberty.tools.eclipse.tests",
+ io.openliberty.tools.eclipse.debug;x-friends:="io.openliberty.tools.eclipse.tests",
  io.openliberty.tools.eclipse.ui.dashboard;x-friends:="io.openliberty.tools.eclipse.tests",
  io.openliberty.tools.eclipse.ui.launch;x-friends:="io.openliberty.tools.eclipse.tests",
  io.openliberty.tools.eclipse.ui.launch.shortcuts;x-friends:="io.openliberty.tools.eclipse.tests",
  io.openliberty.tools.eclipse.ui.preferences;x-friends:="io.openliberty.tools.eclipse.tests"
 Require-Bundle: org.eclipse.ui,
- org.eclipse.equinox.preferences
+ org.eclipse.equinox.preferences,
+ org.eclipse.jdt.debug,
+ org.eclipse.debug.ui,
+ org.eclipse.jdt.launching,
+ org.eclipse.swt,
+ org.eclipse.wst.server.core,
```

Comment:
I'm surprised we have a dependency on `org.eclipse.wst.server.core`.  Are you sure?

Reply:
Yea this must have been leftover from other stuff I was trying out...